app/models/post.rb

```diff
@@ -0,0 +1,16 @@
+class Post < ApplicationRecord
+  belongs_to :users, foreign_key: :author_id, inverse_of: :posts
```

Comment:
- [ ] This association has the following errors as tested:

1. `NameError (uninitialized constant Post::Users):`
    This is because a post only belongs to the user that created it not 'users'.
    The `belongs_to` association should be implemented in singular form i.e `belongs_to :user`
    Besides this, according to the ERD, the foreign key in this Comment table should be **`AuthorID`**, therefore the association should be:
    **`belongs_to :author, class_name: 'User'`**

    Please consider refactoring this appropriately.